### PR TITLE
Release: Adding the i18n messages to the built plugin

### PR DESCRIPTION
**Pull request by youknowriad**
On each release, we generate a new i18n messages file to submit with the plugin. This would allow translating the plugin into other languages.

### CODE REVIEW

**Comment by gziolo** [comment]
We should add `languages/gutenberg.pot` to the list of files ignored by GIT :)

**Review by gziolo — APPROVED** [review]
Looks good, I can see file inside the zipped plugin.

File: bin/build-plugin-zip.sh [hunk 1]
```diff
@@ -79,6 +79,8 @@ status "Installing dependencies..."
 npm install
 status "Generating build..."
 npm run build
+status "Generating translation messages"
```
Comment (aduth) [inline bin/build-plugin-zip.sh]: Nit: Other "in progress" messages end with triple dot ellipsis (`...`).

**Review by aduth — APPROVED** [review]

**Comment by ntwb** [comment]
dotorg no longer requires the `.pot` file, that said I think it should still be included as it allows for custom translations.

Here's the current status of Gutenberg translations: https://translate.wordpress.org/projects/wp-plugins/gutenberg, 7 locales 100% translated, 4 > 90%, and 13 > 80%  🌍 🌎 🌏

**Comment by aduth** [comment]
>dotorg no longer requires the `.pot` file

Could you clarify this? I'd assume if there is tooling, it'd be capable of finding strings within PHP files, but not those in JS.

One issue here is that we're not ever merging the JS strings with the PHP strings, and there are many:

https://github.com/WordPress/gutenberg/search?l=PHP&q=__%28&type=&utf8=%E2%9C%93

This merge step is not yet built out (but will need to be)

**Comment by ntwb** [comment]
@aduth Via https://make.wordpress.org/meta/handbook/documentation/translations/
> How does the translation system import strings? Do I need `.po`/`.pot` files? 
> > The translation system looks through all PHP files in a plugin and finds each gettext call, including `__()` and `_e()`. For this reason, `.po`/`.pot` files are not needed to import, however they’re a great way of ensuring your entire plugin is completely internationalized.

And yes, indeed JS strings are not covered currently, the plan is to add JS support as part of [JavaScript Internationalization](https://make.wordpress.org/core/2016/10/10/javascript-internationalization/) project with a Proof on Concept is [here](https://github.com/ocean90/wp-i18n-tools/pull/1)

**Pull request merged**